core/src/main/java/org/fao/geonet/util/XslUtil.java

```diff
@@ -446,17 +446,25 @@ String twoCharLangCode(String iso3LangCode, String defaultValue) {
             String iso2LangCode = null;
 
             try {
+				final IsoLanguagesMapper mapper = ApplicationContextHolder.get().getBean(IsoLanguagesMapper.class);
```

Comment:
Don't forget to configure your IDE with https://github.com/geonetwork/core-geonetwork/blob/master/.editorconfig. We are using space and not tab for indent. Thanks.

Reply:
Ah, thanks for the heads up - sorry about the formatting. Will bear that in mind for the future.